platform/jewel/markdown/core/src/main/kotlin/org/jetbrains/jewel/markdown/processing/MarkdownProcessor.kt

```diff
@@ -236,7 +236,7 @@ public class MarkdownProcessor(
     }
 
     private fun Node.tryProcessMarkdownBlock(): MarkdownBlock? =
-        // Non-Block children are ignored
+        // Nodes that are not blocks or unsupported types are ignored
```

Comment:
What does "Nodes that are not [...] unsupported types are ignored" mean? I don't think this is clearer than what was here before